protocol/syncer.go

```diff
@@ -305,11 +316,23 @@ func (s *noForkSyncer) Sync(callback func(*types.Block) bool) error {
 
 				continue
 			}
+
+			keys := make([]peer.ID, 0, len(skipList))
+			for i := range skipList {
+				keys = append(keys, i)
+			}
+
+			// remove expired peer
+			for _, id := range keys {
+				if time.Now().Unix() > skipList[id] {
```

Comment:
Better use 1 variable instead of calculating every time.

Reply:
fix it [d42878f](https://github.com/dogechain-lab/dogechain/pull/309/commits/d42878fbe5b4ad5808413d2374915107776c1597)